### PR TITLE
Feat scan result page

### PR DESCRIPTION
**Pull request by PavelFilo**
refactor: moving files into folders
feat: scan result page

### CODE REVIEW

File: app/(app)/scan/scan-result.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      options={{ headerTransparent: true }}
+      actionButton={
+        scanResult === ScanResultEnum.NoViolation ? (
+          <ContinueButton onPress={router.back}>
+            {`${scanResult}.actionButtonLabelBack` as string}
+          </ContinueButton>
+        ) : (
+          <ContinueButton variant="negative" onPress={() => router.push('/offence')}>
+            {`${scanResult}.actionButtonLabel` as string}
+          </ContinueButton>
```
Comment (radoslavzeman) [inline app/(app)/scan/scan-result.tsx:31]: Is this correct? Looks like translation function keys but without the actual `t` function :D 

Also `as string` doesn't look right but I don't see what is the problem why it is needed.

Comment (PavelFilo) [reply app/(app)/scan/scan-result.tsx:31]: It's still todo... texts are going to be changed and I'm also going to add  translations

File: app/(app)/scan/scan-result.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        ) : (
+          <ContinueButton variant="negative" onPress={() => router.push('/offence')}>
+            {scanResult}
+          </ContinueButton>
+        )
+      }
+    >
+      <ContentWithAvatar
+        variant={
+          scanResult === ScanResultEnum.NoViolation
```
Comment (JurajKubrican) [inline app/(app)/scan/scan-result.tsx:35]: Hey, I find [nested ternary operators](https://eslint.org/docs/latest/rules/no-nested-ternary) a bit difficult to follow and prefer to spin them off to explicit functions. something simple like this :)
```
const getResultVariant = (result: ScanResultEnum) => {
  switch (result) {
    case ScanResultEnum.NoViolation:
      return 'success' as const
    case ScanResultEnum.PaasParkingViolationDuplicity:
      return 'warning' as const
    case ScanResultEnum.PaasParkingViolation:
    default:
      return 'error' as const
  }
}
```

**Review by JurajKubrican — APPROVED** [review]
Looks good 👍

**Pull request merged**